spec/models/item_spec.rb

```diff
@@ -90,6 +90,19 @@
       order.item_orders.create(item: @chain, price: @chain.price, quantity: 2)
       expect(@chain.no_orders?).to eq(false)
     end
+
+    it "loaded_image?" do
+      item = create(:item, merchant_id: @bike_shop.id)
+      # result = item.loaded_image?
+
+      # expect(result).to be_truthy
```

Comment:
This test wasn't passing here even though I can see it working on localhost.

Reply:
I had a similar problem with an image-related method... :/

Reply:
Because if it's a model method then it should be testing logic, but if it's a controller method, that is focused on the view, it should exist there.  I am recapping your logic for testing this in the model instead of the item controller.  If that statement ends up being true, then that would explain why you can't test it in the model...it's a controller method.

Reply:
It's an item instance method which is why I'm testing it in the item model spec.